frontend/src/lib/api.ts

```diff
@@ -1206,7 +1218,7 @@ const api = {
             return await new ApiRequest().notebook(notebookId).update({ data })
         },
         async list(): Promise<PaginatedResponse<NotebookType>> {
```

Comment:
We should update the type here given the notebook returned in this case does not have the `content` field